packages/core/src/components/graphs/gauge.ts

```diff
@@ -0,0 +1,397 @@
+// Internal Imports
+import { Component } from "../component";
+import { DOMUtils } from "../../services";
+import {
+	Roles,
+	TooltipTypes,
+	Events,
+	GaugeTypes,
+	Statuses,
+	ArrowDirections
+} from "../../interfaces";
+import { Tools } from "../../tools";
+
+// D3 Imports
+import { select } from "d3-selection";
+import { arc } from "d3-shape";
+
+// arrow paths for delta
+const ARROW_UP_PATH_STRING = "4,10 8,6 12,10";
+const ARROW_DOWN_PATH_STRING = "12,6 8,10 4,6";
+
+export class Gauge extends Component {
+	type = "gauge";
+
+	// We need to store our arcs so that addEventListeners() can access them
+	arc: any;
+	backgroundArc: any;
+
+	init() {
+		const eventsFragment = this.services.events;
+	}
+
+	getValue(): number {
+		const data = this.model.getData();
+		const value = data.find((d) => d.group === "value")?.value ?? null;
+		return value;
+	}
+
+	getValueRatio(): number {
+		const value = Tools.clamp(this.getValue(), 0, 100);
+		return value / 100;
+	}
+
+	getDelta(): number {
+		const data = this.model.getData();
+		const delta = data.find((d) => d.group === "delta")?.value ?? null;
+		return delta;
+	}
+
+	getArcRatio(): number {
+		const options = this.model.getOptions();
+		const type = Tools.getProperty(options, "gauge", "type");
+		const arcRatio = type === GaugeTypes.FULL ? 1 : 0.5;
+		return arcRatio;
+	}
+
+	getArcSize(): number {
+		return this.getArcRatio() * Math.PI * 2;
+	}
+
+	getStartAngle(): number {
+		const arcSize = this.getArcSize();
+		if (arcSize === 2 * Math.PI) {
+			return 0;
+		}
+		return -arcSize / 2;
+	}
+
+	// use provided arrow direction or default to using the delta
+	getArrow(delta): string {
+		const options = this.model.getOptions();
+		const arrowDirection = Tools.getProperty(
+			options,
+			"gauge",
+			"arrowDirection"
+		);
+
+		switch (arrowDirection) {
+			case ArrowDirections.UP:
+				return ARROW_UP_PATH_STRING;
+			case ArrowDirections.DOWN:
+				return ARROW_DOWN_PATH_STRING;
+			default:
+				return delta > 0
```

Comment:
is there a way to disable the arrow?

Reply:
I can add in a flag 👍 
I believe the thought process was just **no delta = no chevron** but if theres a delta then there should be a chevron (to atleast make it known that this is a delta dimension) but we can leave this up to the user

Reply:
ok, we can ship it this way and see if there are any requests for removing the arrow